QUEST_LV_0100.tsv

```diff
@@ -683,19 +683,19 @@ QUEST_LV_0100_20150317_000682	Even if we fail preventing the disease, we would b
 QUEST_LV_0100_20150317_000683	You got it. Thank you so much.{nl}You saved lots of lives in Fedimian.
 QUEST_LV_0100_20150317_000684	{memo X}We can suppress the vicious energy if the magic of the tomb gets unleashed.{nl}I need the activating stone that is located inside the protector.
 QUEST_LV_0100_20150317_000685	The manual for the secret place
-QUEST_LV_0100_20150317_000686	I hid the secret letters for the revelator who would come someday.{nl}Find the hidden energy of the protector and open the secret door.
+QUEST_LV_0100_20150317_000686	$We hid the secret treasures for the Revelator who would come someday.{nl}Find and absorb the protectors' hidden energy to open the secret door.
 QUEST_LV_0100_20150317_000687	The hiding place of the brother I used to serve is located at the abondoned farm up there.{nl}Please get the diary of the brother there.
 QUEST_LV_0100_20150317_000688	Grita
 QUEST_LV_0100_20150317_000689	{memo X}It seems that there are more monsters here now compared to the number monsters that were present before I lost my conciousness.{nl}Did something happen to Gabija?
 QUEST_LV_0100_20150317_000690	{memo X}It may be better for Gabija to let her know that we are here.{nl}There's a pole that we can ignite light on. Let's go.
 QUEST_LV_0100_20150317_000691	{memo X}Good. Since she knows that we came, she will recover her health.{nl}Let's go up to see Gabija.{nl}{nl}
-QUEST_LV_0100_20150317_000692	The protector of the secret letters of the kingdom
-QUEST_LV_0100_20150317_000693	You are qualified to receive the secret letters.
+QUEST_LV_0100_20150317_000692	$Protector of the Kingdom's Secret Treasures
```

Comment:
Protector of the Kingdom's Secret Treasures -> Guarding the Kingdom's Secret Treasures